src/main/scala/nl/vroste/zio/kinesis/client/DynamicConsumerLive.scala

```diff
@@ -104,13 +104,13 @@ private[client] class DynamicConsumerLive(
       }
 
       override def leaseLost(leaseLostInput: LeaseLostInput): Unit =
-        shardQueue.stop("lease lost")
+        if (shardQueue != null) shardQueue.stop("lease lost")
```

Comment:
could we have `shardQueue` as an Option to avoid `null` ?

Reply:
Done!